src/DIRAC/WorkloadManagementSystem/Service/MatcherHandler.py

```diff
@@ -56,7 +56,7 @@ def export_requestJob(self, resourceDescription):
 
         resourceDescription["Setup"] = self.serviceInfoDict["clientSetup"]
         credDict = self.getRemoteCredentials()
-        pilotRef = resourceDescription.get("PilotReference", "Unknown")
+        pilotRef = resourceDescription.get("PilotReference")
```

Comment:
This was not harming.

Reply:
Then, what's the point of having the following piece of code here?

https://github.com/DIRACGrid/DIRAC/pull/7924/files#diff-6e5904ade77b6cd702bb41532101e8027ba0ebc3a3430c98332efd246df5af5eR55-R56

Reply:
It was probably to handle cases of old pilots without a pilotRef.

Reply:
Thanks for the clarification, I reverted my change.